ci-operator/step-registry/openshift-qe/node-density/openshift-qe-node-density-ref.yaml

```diff
@@ -13,5 +13,14 @@ ref:
     requests:
       cpu: 100m
       memory: 100Mi
+  env:
+  - name: PODS_PER_NODE
+    default: "150"
```

Comment:
We should set the default to the PODS_PER_NODE that is the same as the previous version (230) to not change the tests that were running on non rosa versions and set the PODS_PER_NODE as an environment variable for rosa to 150 in the top level config